includes/unicode.inc

```diff
@@ -334,6 +334,10 @@ function _mime_header_decode($matches) {
  *   The input $text, with all HTML entities decoded once.
  */
 function decode_entities($text, $exclude = array()) {
+  if (empty($exclude) && version_compare(phpversion(), '5.0') >= 0) {
+    return html_entity_decode($text, ENT_QUOTES, 'UTF-8');
```

Comment:
I wonder if this bit is worth including? It might even be a little faster...

Reply:
I see it's what's on D7. I don't know.. Being picky about it it could be that the output of the regex is slightly different under a certain circumstance and we would be changing it's functionality.

If I would have to cast a vote, I'd say no.. :)